Automattic/vip-go-mu-plugins
# Update elasticpress-next to 82261dc4bf3bb83d333819396c7d9d2257d93a91

## Description

This includes a fix for SyncManager::tear_down to also remove VIP-specific hooks

## Changelog Description

### Plugin Updated: Enterprise Search

Disable VIP-specific hooks ElasticPress\Indexable\Post\SyndManager::tear_down()

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

This needs #4295 for testing

1. Enable Search but don't add index.
2. Perform a variety of publishing activity - make sure index was not created
3. Edit some terms - make sure the queue stays empty.
